src/store/singing.ts

```diff
@@ -746,6 +748,59 @@ export const singingStore = createPartialStore<SingingStoreTypes>({
       dispatch("RENDER");
     },
   },
+  SET_VOLUME_EDIT_DATA: {
+    // ボリューム編集データをセットする。
+    // track.volumeEditDataの長さが足りない場合は、伸長も行う。
+    mutation(state, { volumeArray, startFrame, trackId }) {
+      const track = getOrThrow(state.tracks, trackId);
+      const volumeEditData = track.volumeEditData;
+      const tempData = [...volumeEditData];
+      const endFrame = startFrame + volumeArray.length;
+      if (tempData.length < endFrame) {
+        const valuesToPush = new Array(endFrame - tempData.length).fill(
+          VALUE_INDICATING_NO_DATA,
+        );
+        tempData.push(...valuesToPush);
+      }
+      tempData.splice(startFrame, volumeArray.length, ...volumeArray);
+      track.volumeEditData = tempData;
+    },
+    async action({ dispatch, commit }, { volumeArray, startFrame, trackId }) {
+      if (startFrame < 0) {
+        throw new Error("startFrame must be greater than or equal to 0.");
+      }
+      if (!isValidVolumeEditData(volumeArray)) {
+        throw new Error("The pitch edit data is invalid.");
+      }
+      commit("SET_VOLUME_EDIT_DATA", { volumeArray, startFrame, trackId });
+
+      dispatch("RENDER");
+    },
+  },
+
+  ERASE_VOLUME_EDIT_DATA: {
+    mutation(state, { startFrame, frameLength, trackId }) {
+      const track = getOrThrow(state.tracks, trackId);
+      const volumeEditData = track.volumeEditData;
+      const tempData = [...volumeEditData];
+      const endFrame = Math.min(startFrame + frameLength, tempData.length);
+      tempData.fill(VALUE_INDICATING_NO_DATA, startFrame, endFrame);
+      track.volumeEditData = tempData;
+    },
+  },
+
+  CLEAR_VOLUME_EDIT_DATA: {
+    // ピッチ編集データを失くす。
+    mutation(state, { trackId }) {
+      const track = getOrThrow(state.tracks, trackId);
+      track.pitchEditData = [];
+    },
```

Comment:
ここも`ピッチ`、`pitchEditData`になってますね。
```suggestion
  CLEAR_VOLUME_EDIT_DATA: {
    // ボリューム編集データを失くす。
    mutation(state, { trackId }) {
      const track = getOrThrow(state.tracks, trackId);
      track.volumeEditData = [];
    },
```